diff --git a/node_modules/@vscode/spdlog/deps/spdlog/include/spdlog/fmt/bundled/core.h b/node_modules/@vscode/spdlog/deps/spdlog/include/spdlog/fmt/bundled/core.h
index 8f1a6f1..b7a5f0c 100644
--- a/node_modules/@vscode/spdlog/deps/spdlog/include/spdlog/fmt/bundled/core.h
+++ b/node_modules/@vscode/spdlog/deps/spdlog/include/spdlog/fmt/bundled/core.h
@@ -263,7 +263,7 @@
 #  define FMT_USE_CONSTEVAL 0
 #endif
 #if FMT_USE_CONSTEVAL
-#  define FMT_CONSTEVAL consteval
+#  define FMT_CONSTEVAL constexpr
 #  define FMT_CONSTEXPR20 consteval
 #else
 #  define FMT_CONSTEVAL
